contentcuration/contentcuration/serializers.py

```diff
@@ -762,11 +762,27 @@ class PublicChannelSerializer(ChannelFieldMixin, serializers.ModelSerializer):
     """
     kind_count = serializers.SerializerMethodField('generate_kind_count')
     matching_tokens = serializers.SerializerMethodField('match_tokens')
+    icon_encoding = serializers.SerializerMethodField('get_thumbnail_encoding')
```

Comment:
If we decide it is safe to always return thumbnail_encoding, we could simplify this to:
`icon_encoding = serializers.JSONField(source='thumbnail_encoding')`